app/src/main/java/fr/gaulupeau/apps/Poche/ui/ReadArticleActivity.java

```diff
@@ -746,7 +746,13 @@ private String readRawString(int id) throws IOException {
             return sb.toString();
         } finally {
             if(reader != null) {
-                reader.close();
+                try {
+                    reader.close();
+                } catch (IOException e) {
+                    // unrecoverable exception, only log it and move on
+                    Log.w(TAG, "readRawString() failed to close reader\n" + e.toString()
+                            + "\n" + e.getStackTrace().toString());
+                }
```

Comment:
We close reader for an internal resource here, any errors are unlikely. I'd recommend `catch(IOException ignored) {}`